source/work-with-indexes.txt

```diff
@@ -19,15 +19,15 @@ Work with Indexes
 
 .. toctree::
 
-   /indexes/single-field-index.txt
-   /indexes/compound-index.txt
-   /indexes/multikey-index.txt
-   /indexes/atlas-search-index.txt
-   /indexes/text-index.txt
-   /indexes/geospatial-index.txt
-   /indexes/unique-index.txt
-   /indexes/wildcard-index.txt
-   /indexes/clustered-index.txt
+   Single-Field Indexes </indexes/single-field-index.txt>
```

Comment:
Did Lauren say something about whether or not to keep the word "Indexes" in cases like this? I just don't remember what she said

Reply:
Yeah, when the word appears in the next level heading we'd want to leave it out.  So all of these could drop "Indexes."